vars/uploadArtifacts.groovy

```diff
@@ -2,6 +2,9 @@ void call(Map args = [:]) {
     def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
 
     def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: args.manifest))
+    def fileLocation = buildManifest.components.get("OpenSearch").artifacts.get("dist").first()
```

Comment:
This is the min artifact location, so probably `def minArtifactPath`?

Reply:
Changed